packages/core/admin/admin/src/pages/Roles/ListPage/tests/index.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * Tests for ListPage
+ *
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import ListPage from '../index';
+
+describe('<ListPage />', () => {
+  it('renders and matches the snapshot', () => {
+    const {
+      container: { firstChild },
+    } = render(<ListPage />);
+
+    expect(firstChild).toMatchInlineSnapshot();
```

Comment:
This test match nothing 🤔 . I think you should generate the snapshot here :)

Reply:
It's because the test if failing, you need to wrap the ListPage in the `<IntProvider />` and the `<ThemeProviders />` components